api/src/opentrons/protocol_api/core/engine/protocol.py

```diff
@@ -393,6 +400,26 @@ def get_module_cores(self) -> List[ModuleCore]:
         """Get all loaded module cores."""
         return list(self._module_cores_by_id.values())
 
+    def define_liquid(
+        self,
+        name: str,
+        description: Optional[str],
+        display_color: Optional[str],
+    ) -> Liquid:
+        """create a liquid to load into a well."""
+        loaded_liquid = self._engine_client.add_liquid(
+            name=name, description=description, color=display_color
+        )
+
+        return Liquid(
+            _id=loaded_liquid.id,
+            name=loaded_liquid.displayName,
+            description=loaded_liquid.description,
+            display_color=loaded_liquid.displayColor.__root__
+            if loaded_liquid.displayColor
+            else None,
```

Comment:
Please use parentheses for readability when doing these `if/else` assignments. I think newer `black` will add this formatting

```suggestion
            display_color=(
                loaded_liquid.displayColor.__root__
                if loaded_liquid.displayColor
                else None
            ),
```